tests/unit/test_event_stream.py

```diff
@@ -14,14 +14,14 @@ def collect_events(stream):
 @pytest.mark.asyncio
 async def test_basic_flow():
```

Comment:
These tests no longer need to be async I suppose?

Reply:
Thanks!